system_core/metadata.csv

```diff
@@ -1,5 +1,5 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
 system.core.count,gauge,,core,,The number of CPU cores on the host,0,system,cpu cores,
-system.core.user,gauge,,second,,A given core's user-space CPU time,0,system,cpu user,
-system.core.system,gauge,,second,,A given core's system CPU time,0,system,cpu system,
-system.core.idle,gauge,,second,,A given core's idle CPU time,0,system,cpu idle,
+system.core.user,gauge,,percent,,A given core's user-space CPU time percentage,0,system,cpu user,
```

Comment:
Could we maybe reword this a bit? 

`The percentage of time a given core spent in user mode/system mode/idle` maybe?

https://psutil.readthedocs.io/en/latest/
user: time spent by normal processes executing in user mode; on Linux this also includes guest time
system: time spent by processes executing in kernel mode
idle: time spent doing nothing

Reply:
Thanks! that's a lot clearer, I'll make that change